src/App.tsx

```diff
@@ -180,8 +188,23 @@ const App = () => {
                   handleSubmit={handleProposal("coreEvalProposal")}
                   titleDescOnly={false}
                   title="/agoric.swingset.CoreEvalProposal"
-                  description="This is a governance proposal that executes code. You will need to provide a JS Bundle, and a JSON Permit file."
                   msgType="coreEvalProposal"
+                  description={
+                    <>
+                      This is a governance proposal that executes code after a
+                      passing vote. The JSON Permit grants{" "}
+                      <Link href="https://docs.agoric.com/guides/coreeval/permissions.html">
+                        capabilities
+                      </Link>{" "}
+                      and the JS Script can install and start a contract. These
```

Comment:
```suggestion
and the JS Script can start a contract. These
```
`Instantiate` is probably the correct word. But less words is better? Also leaving out "... or update"